### PR TITLE
CNVCu0 Workload

### PR DESCRIPTION
**Pull request by ebenahar**
This test runs for around 5 minutes, during this time it does the following:
1) starts an FIO pod to run in the background throughout the test execution
2) starts measuring the latency in the background throughout the test execution
3) creates, deletes PVCs and pods of block and file, write IO with data validation, and measures the creation and deletion of each
4) once 5 minutes are over, it compares the p99 latency, pvc creation time and pvc deletion time to the accepted p99 latency, pvc creation and pvc deletion we define as the benchmark. In case any of them is higher, the test fails

### CODE REVIEW

**Comment by vasukulkarni** [comment]
@ebenahar I think you can reuse things from here with few additions instead of new: https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/scale/test_pv_scale_ocs_create_delete_pvcs.py#L174-L182

File: ocs_ci/ocs/resources/pod.py [hunk 1]
```diff
@@ -298,6 +298,8 @@ def run_io(
         fio_filename=None,
         bs="4K",
         end_fsync=0,
+        verify=None,
```
Comment (mykaul) [inline ocs_ci/ocs/resources/pod.py:301]: But we do wish to perform data verification...

File: ocs_ci/ocs/resources/pod.py [hunk 2]
```diff
@@ -298,6 +298,8 @@ def run_io(
         fio_filename=None,
         bs="4K",
         end_fsync=0,
```
Comment (mykaul) [inline ocs_ci/ocs/resources/pod.py:300]: If we are not using direct IO and we are not using fsync, we can't be sure data has landed fully. I suggest using direct IO.

**Comment by ebenahar** [comment]
> @ebenahar I think you can reuse things from here with few additions instead of new: https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/scale/test_pv_scale_ocs_create_delete_pvcs.py#L174-L182

@vasukulkarni thanks for the reference, I will check that

**Comment by ebenahar** [comment]
had to re-create the PR because of the branch name - https://github.com/red-hat-storage/ocs-ci/pull/3978

**Pull request closed without merging**